src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -576,14 +576,14 @@ final case class ProcessApiServiceImpl(
     contexts: Seq[(String, String)],
     toEntityMarshallerEServiceConsumers: ToEntityMarshaller[EServiceConsumers],
     toEntityMarshallerProblem: ToEntityMarshaller[Problem]
-  ): Route = authorize(ADMIN_ROLE, API_ROLE, SECURITY_ROLE, M2M_ROLE) {
+  ): Route = authorize(ADMIN_ROLE, API_ROLE, SECURITY_ROLE, M2M_ROLE, SUPPORT_ROLE) {
     val operationLabel =
       s"Retrieving consumers for EService $eServiceId"
     logger.info(operationLabel)
 
     val result: Future[EServiceConsumers] = for {
       result <- ReadModelQueries.listConsumers(eServiceId, offset, limit)(readModel)
-      apiResults = result.results.map(_.toApi)
+      apiResults = result.results.map(_.toApi).sortBy(r => (r.descriptorVersion, r.consumerName.toLowerCase()))
```

Comment:
this would sort the current page, but not based on the whole result set.
Sorting should be done only in the query